uber/models/attendee.py

```diff
@@ -230,11 +231,11 @@ class Attendee(MagModel, TakesPaymentMixin):
     paid = Column(Choice(c.PAYMENT_OPTS), default=c.NOT_PAID, index=True, admin_only=True)
     overridden_price = Column(Integer, nullable=True, admin_only=True)
     base_badge_price = Column(Integer, default=0, admin_only=True)
-    amount_paid = Column(Integer, default=0, admin_only=True)
+    amount_paid_override = Column(Integer, default=0, admin_only=True)
```

Comment:
We're keeping these columns for now just to help make sure everything is being tracked properly at some level. They aren't referred to in the code and can be removed later if we decide we don't need them as a backup.